src/v1/sources/shopify/transform.js

```diff
@@ -13,7 +15,12 @@ const process = async (inputEvent) => {
     const responseV2 = await processEventFromPixel(event);
     return responseV2;
   }
-  // this is for common logic for server-side events processing for both pixel and tracker apps.
+  if (query_parameters && query_parameters?.version?.[0] === 'pixel') {
```

Comment:
are query_parameters values getting sent in array type from rudder-server ? cc: @vinayteki95